CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 Changelog
 ================================================================================
 
+2307-1
+--------------------------------------------------------------------------------
+
+### Main Changes
+
+* Updated dependencies to CoreMedia Content Cloud v11.2307.1.
+* `DownloadFromGlobalLinkAction` now uses the newly introduced API to
+  execute the XLIFF import as the robot user.
+* When upgrading, you will have to upload workflow definition again to benefit
+    from the latest changes:
+
+    * In v11.2307.1 the `CleanInTranslationFinalAction` is now taking care of
+      cleaning up so called _mergeVersions_ on the content. These marked were 
+      previously only removed asynchronously after aborting the workflow. This
```

Comment:
Don't get the meaning of "These marked were previously only removed asynchronously after aborting the workflow." Can you rephrase (or maybe fix a typo) to make it more clear?